data_structures/arrays/equilibrium_index_in_array.py

```diff
@@ -0,0 +1,59 @@
+"""
+Find the Equilibrium Index of an Array.
+Reference: https://www.geeksforgeeks.org/equilibrium-index-of-an-array/
+
+Python doctests can be run with the following command:
+python -m doctest -v equilibrium_index.py
+
+Given a sequence arr[] of size n, this function returns
+an equilibrium index (if any) or -1 if no equilibrium index exists.
+
+The equilibrium index of an array is an index such that the sum of
+elements at lower indexes is equal to the sum of elements at higher indexes.
+
+
+
+Example Input:
+arr = [-7, 1, 5, 2, -4, 3, 0]
+Output: 3
+
+"""
+
+
+def equilibrium_index(arr: list[int], size: int) -> int:
+    """
+    Find the equilibrium index of an array.
+
+    Args:
+        arr (List[int]): The input array of integers.
+        n (int): The size of the array.
```

Comment:
Please do not reference types within docstrings

Reply:
done

Reply:
Review needed!!

Reply:
> Please do not reference types within docstrings

Done!!
Review need to merge